src/cli/CommandLineStatisticsPresenterConfig.ts

```diff
@@ -0,0 +1,18 @@
+export interface CommandLineStatisticsPresenterConfig {
```

Comment:
deliberately DIDNT look into the presenter/cmdline stuff: they don't belong to this huge PR. please split)

Reply:
why not?

Reply:
cool. don't.